autoload/lsp/tag.vim

```diff
@@ -0,0 +1,155 @@
+let s:tag_kind_priority = ['definition', 'declaration', 'implementation', 'type definition']
+
+function! s:not_supported(what) abort
+    call lsp#log(a:what . ' not supported for ' . &filetype)
+endfunction
+
+function! s:location_to_tag(loc) abort
+    if has_key(a:loc, 'targetUri')
+        let l:uri = a:loc['targetUri']
+        let l:range = a:loc['targetRange']
+    else
+        let l:uri = a:loc['uri']
+        let l:range = a:loc['range']
+    endif
+
+    if !lsp#utils#is_file_uri(l:uri)
+        return v:null
+    endif
+
+    return {
+        \ 'filename': lsp#utils#uri_to_path(l:uri),
+        \ 'cmd': string(l:range['start']['line'] + 1)
```

Comment:
Can you instead use functions from https://github.com/prabirshrestha/vim-lsp/blob/master/autoload/lsp/utils/position.vim so it works for multi byte chars.